practice-app/app/comment.py

```diff
@@ -87,4 +87,9 @@
     "x-rapidapi-key": "c4ab16012amsh73b5a257264eb3dp11ade4jsnb69ec0b79098",
     "x-rapidapi-host" :"google-search3.p.rapidapi.com"
 }
+@comment_api.route("/api/v1.0/events/<id:event_id>/comments",methods=["GET"])
+def getComment(event_id):
+    
+    commentlist=session.query(Comment).filter(Comment.postID==event_id).all()
+    return jsonify({"comment":list(commentlist)})
```

Comment:
You need to manually serialize Comment object in this way since SQLAlchemy table objects are not serializable. 
```suggestion
    return jsonify([{col.name: str(getattr(comment, col.name)) for col in comment.__table__.columns} for comment in commentlist]), 200 
```